dist/formats/specialist_document/frontend/schema.json

```diff
@@ -46,24 +46,25 @@
     "document_type": {
       "type": "string",
       "enum": [
+        "aaib_report",
+        "asylum_support_decision",
+        "business_finance_support_scheme",
         "cma_case",
-        "employment_appeal_tribunal_decision",
```

Comment:
There's a bunch of seemingly-unrelated changes here.  Is this an automatically generated file we've just not kept up to date?

Reply:
I think that's just making this list alphabetical?